packages/codemod/plop/component/transform-remove-props.hbs

```diff
@@ -0,0 +1,26 @@
+import {
+  wrap,
+  getCoreImportsForFile,
+  getComponentNameOrAliasFromImports,
+  findComponentElements,
+  removeProp
+} from "../../../src/utils";
+import { TransformationContext } from "../../../types";
+
+/**
+ * 1. TODO: What this codemod does?
```

Comment:
Why the numeric list? Is there another thing to do?

Reply:
on some of the codemods we're doing more than one action
as we continute with Vibe3, probably most of the codemods would include more than one action